### ISSUE TITLE
The 'silent' keyword argument does not behave pythonically in version 2.1.0

### ISSUE THREAD

**Issue by tddpirate** [chezou/tabula-py#246]
# Summary of your issue

When I give `silent` as the last argument to `tabula.read_pdf()`, the mere existence of `silent` keyword causes it to be considered as `True`. To unsilence Tabula's `sys.stderr` output, I have to omit it. Setting `silent=False` or `silent=None` did not turn on Tabula's `sys.stderr` output.
I did not test other Tabula functions or other `tabula.io.build_options()` options, but I suspect that the bug exists also for them.

Workaround which worked for me:
```
tabula.read_pdf(...., **({ "silent": True } if silent else {})
```
which is not Pythonic.

# Check list before submit

- [x] Did you read [FAQ](https://tabula-py.readthedocs.io/en/latest/faq.html)?

- [ ] (Optional, but really helpful) Your PDF URL: ?
Not PDF-specific.

- [X] Paste the output of `import tabula; tabula.environment_info()` on Python REPL: ?
>>> import tabula
>>> tabula.environment.info()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'tabula' has no attribute 'environment'
>>> tabula.environment_info()
Python version:
    3.8.2 (default, Apr  2 2020, 08:26:09) 
[GCC 8.3.0]
Java version:
    openjdk version "11.0.7" 2020-04-14
OpenJDK Runtime Environment (build 11.0.7+10-post-Debian-3deb10u1)
OpenJDK 64-Bit Server VM (build 11.0.7+10-post-Debian-3deb10u1, mixed mode, sharing)
tabula-py version: 2.1.0
platform: Linux-4.19.0-9-amd64-x86_64-with-glibc2.28
uname:
    uname_result(system='Linux', node='c4', release='4.19.0-9-amd64', version='#1 SMP Debian 4.19.118-2+deb10u1 (2020-06-07)', machine='x86_64', processor='')
linux_distribution: ('Debian GNU/Linux', '10', 'buster')
mac_ver: ('', ('', '', ''), '')

# What did you do when you faced the problem?
I used the workaround:
```
tabula.read_pdf(...., **({ "silent": True } if silent else {})
```

## Code:

```
tabula.read_pdf(...)
```
with the following values of `silent` option: 
- option does not exist
- `silent=True`
- `silent=False`
- `silent=None`

## Expected behavior:
- missing option: messages about font errors are outputted.
- `silent=True`: messages about font errors are NOT outputted.
- `silent=False`: messages about font errors are outputted.
- `silent=None`: messages about font errors are outputted.

## Actual behavior:
- missing option: messages about font errors are outputted - OK
- `silent=True`: messages about font errors are NOT outputted - OK
- `silent=False`: messages about font errors are NOT outputted - NOT OK
- `silent=None`: messages about font errors are NOT outputted - NOT OK

## Related Issues:
The bug is cosmetic by its nature - does not affect real functionality and has an easy workaround.

**Comment by chezou** [comment]
@tddpirate Thanks for reporting. I understood what you mean. The option itself is passed to tabula-java respectively but this workaround doesn't respect the value of `silent` option itself.
https://github.com/chezou/tabula-py/blob/72c60f65f779d3e56d40bb3b9d14e442ac44b5c4/tabula/io.py#L65

It should be treated as

```
if options.get("silent"):
```

I think it's an easy task to fix, so I keep open this issue for the first contributor.

**Comment by chezou** [comment]
Fixed by #254
